plugins/mcreator-localization/help/default/particle/always_show.md

```diff
@@ -0,0 +1 @@
+If checked, this particle will be rendered even if it's not in range for render.
```

Comment:
Not true. This parameter controls if the particle is shown even if the particles are set to minimal.

Reply:
:heavy_check_mark: Fixed